frappe/core/doctype/user/user.py

```diff
@@ -1382,3 +1420,20 @@ def impersonate(user: str, reason: str):
 	notification.set("type", "Alert")
 	notification.insert(ignore_permissions=True)
 	frappe.local.login_manager.impersonate(user)
+
+
+@frappe.whitelist()
+@rate_limit(limit=10, seconds=60 * 60, methods="POST")
+def clear_session(sid_hash: str):
```

Comment:
add permission check - either same user or System Manager?

also - do we want to allow clearing the current session - will it be accompanied by logging you out in the UI?